install.sh

```diff
@@ -71,15 +98,24 @@ pacman --noconfirm -S \
 	python \
 	vulkan-icd-loader \
 	lib32-vulkan-icd-loader \
-	vulkan-radeon \
-	lib32-vulkan-radeon \
-	steam
+	steam \
+	steam-native-runtime
```

Comment:
since we are not using this by default, no point in installing this. if someone wants to use it, they can install manually and modify the steamos session. i am open to using the native runtime by default, but when i tested this the majority of my games wouldnt even start.

Reply:
I removed it from project after some research, maybe in future it will be much better to use !